{"filter":false,"title":"simple.py","tooltip":"/my_package/simple.py","undoManager":{"mark":2,"position":2,"stack":[[{"start":{"row":0,"column":0},"end":{"row":5,"column":51},"action":"insert","lines":["#! /usr/bin/env python","","import rospy","","rospy.init_node('ObiWan')","print \"Help me Obi-Wan Kenobi, you're my only hope\""],"id":1}],[{"start":{"row":5,"column":50},"end":{"row":5,"column":51},"action":"remove","lines":["\""],"id":2}],[{"start":{"row":5,"column":50},"end":{"row":5,"column":51},"action":"insert","lines":["\""],"id":3}]]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":5,"column":51},"end":{"row":5,"column":51},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1537418594746,"hash":"cff05e149ed4c0c36faec9d5e54b72dd6ae45617"}